src/Maker/MakeAuthenticator.php

```diff
@@ -346,5 +346,10 @@ public function configureDependencies(DependencyBuilder $dependencies, InputInte
             Yaml::class,
             'yaml'
         );
+
+        $dependencies->addClassDependency(
+            TwigBundle::class,
+            'twig-bundle'
+        );
```

Comment:
I think this isn't right... it's more complicated than this. This command could be used to generate a blank authenticator - no Twig needed there. Twig IS needed if we generate the login form authenticator. We have a few examples in the code of doing this check manually in a command based on user input.